src/models/ExclusiveEditMode.php

```diff
@@ -202,4 +204,14 @@ private function extendLockTime(): void
             }
         }
     }
+
+    /**
+     * add lock timeout to dataArr for clarity, indicating until when the entity remains locked.
+     */
+    private function addLockTimeout(string $lockedAt): void
```

Comment:
this is a one line private function used only once, you can remove it and move its body into its call site